mne/annotations.py

```diff
@@ -214,6 +214,25 @@ def __iadd__(self, other):
                                                  other.orig_time))
         return self.append(other.onset, other.duration, other.description)
 
+    def __iter__(self):
+        """Iterate over the annotations."""
+        for idx in range(len(self.onset)):
+            yield self.__getitem__(idx)
+
+    def __getitem__(self, key):
+        """Propagate indexing and slicing to the underlying numpy structure."""
+        if isinstance(key, int):
+            out_keys = ('onset', 'duration', 'description', 'orig_time')
+            out_vals = (self.onset[key], self.duration[key],
+                        self.description[key], self.orig_time)
+            return OrderedDict(zip(out_keys, out_vals))
+        else:
+            key = list(key) if isinstance(key, tuple) else key
+            return Annotations(onset=self.onset[key].copy(),
+                               duration=self.duration[key].copy(),
+                               description=self.description[key].copy(),
```

Comment:
... so no need for any of these copy calls, because a copy is made in `Annotations.__init__`